modules/ROOT/pages/instanton.adoc

```diff
@@ -338,6 +338,36 @@ When you deploy to Kubernetes, the container must be granted the `CHECKPOINT_RES
             - ALL
 ----
 
+==== Red Hat OpenShift security context constraints
+
+To deploy applications to Red Hat OpenShift with InstantOn, you must specify a security context constraint (SCC) that at a minimum grants the capabilities to the defaults that are needed for InstantOn. The following example defines an SCC with these capabilities that is called `defaultAddCapabilities`:
```

Comment:
This doesn't read right to me.
> that at a minimum grants the capabilities to the defaults that are needed for InstantOn.

The capabilities listed under `defaultAddCapabilities` are adding the capabilities to the set of capabilities which are granted to any pod.  So it is adding capabilities to that list of default capabilities.  To me it would make more sense to read like this:

> that at a minimum specifies a list of additional capabilities that are added to any pod. The following example defines an SCC with the required capabilities by using the parameter called `defaultAddCapabilities`: